monkey/infection_monkey/monkey.py

```diff
@@ -180,14 +186,27 @@ def _setup(self):
         control_channel.register_agent(self._opts.parent)
 
         config = control_channel.get_config()
-        self._monkey_inbound_tunnel = self._control_client.create_control_tunnel(
-            config.keep_tunnel_open_time
+
+        local_port = get_free_tcp_port()
+        sock, ip_str, port = connect([self._opts.server])
```

Comment:
We don't have `self._opts.server` anymore. We have list of servers.

Reply:
Thanks!